core/src/main/java/com/graphhopper/routing/VirtualEdgeIterator.java

```diff
@@ -135,6 +132,12 @@ public EdgeIteratorState setName( String name )
     }
 
     @Override
+    public boolean getBoolean(int key, boolean _default, PMap kwargs)
```

Comment:
Now I'm confused. Why do we need to specify a _default? I would implicitly assume that every key defines its own defaults indirectly.